website/docs/reference/seed-properties.md

```diff
@@ -7,12 +7,12 @@ title: Seed properties
 </Changelog>
 
 Seed properties can be declared in `.yml` files in:
-- your `data/` directory (as defined by the [`data-paths` config](data-paths))
-- your `models/` directory (as defined by the [`source-paths` config](source-paths)), for backwards compatibility
+- your `seeds/` directory (as defined by the [`seed-paths` config](seed-paths))
```

Comment:
- a changelog entry for this one feels justified
- [as above/elsewhere] these properties can really be defined anywhere, we should recommend defining them in the `seeds/` directory (as we do below)